src/main/java/com/djrapitops/extension/FactionsUUIDExtension.java

```diff
@@ -127,8 +142,16 @@ public double powerMax(UUID playerUUID) {
     )
     @Conditional("hasFaction")
     public String role(UUID playerUUID) {
-        return getFPlayer(playerUUID)
-                .getRole().getTranslation().format();
+        if (getTranslation == null) { // FUUID 0.5.25+
+            return getFPlayer(playerUUID)
+                    .getRole().getTranslation();
+        } else {
+            try {
+                return ((TL) getTranslation.invoke(getFPlayer(playerUUID))).format(); // FUUID prior to 0.5.25, and forks.
+            } catch (IllegalAccessException | InvocationTargetException ignored) {
+                return ""; // Shouldn't be possible to get here.
```

Comment:
Added a code suggestion that ensures no bad data ends up in the database
```suggestion
                throw new NotReadyException();
```